resources/web/README.md

```diff
@@ -1,15 +1,69 @@
+---    
+sidebar: auto
+---      
+
 # Web Development
 
-## How do I get started with web development
+## Frontend Development
+<LevelWithButton desc="Frontend Development is the process of implementing user interfaces for web applications. As a frontend developer, you create the portion of the website that people interact with and ensure that the design requirements for how your application should 'look' and 'feel' are met" link="https://frontendmasters.com/books/front-end-handbook/2018/what-is-a-FD.html" button="Learn more about it here!" image="https://invexi.com/wp-content/uploads/2015/06/frontend.jpg"/>
```

Comment:
The Level blurbs should be short, and they should describe the resource you're linking. The text you've written is solid, and can go under the heading.